apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx

```diff
@@ -679,7 +679,6 @@ export const OrganizationExternalLinks: React.FC<
                 pureChildren={true}
               >
                 <Button
-                  as="a"
```

Comment:
Try tabbing through the page, when we skip the `as="a"` then need to tab twice to proceed to the next element (because we have an <a> tag and a nested <button> tag.

Try adding unfocusable={true] and see if that makes it right

Reply:
Seems to work fine with the `unfocusable` addition